lib/view.js

```diff
@@ -267,3 +267,43 @@ export function reactVirtualizedToReactVirtualized(view) {
     , columnWidth
   }
 }
+
+/**
+  * Ensure dates and times in resources are in ISO format.
+  * @param {resource}
+  * @return {resource} with formatted dates and times
+  */
+export function normalizeDateAndTime(resource) {
```

Comment:
@zelima we are writing in ES6 in `/lib/` and compiling to ES5 in `/dist/` so they look almost identical.